db/migrate/20220407062906_create_orders.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :orders do |t|
+      t.references :user, foreign_key: true
+      t.references :payment_method, foreign_key: true
+      t.text :status
```

Comment:
status sao lại text hè, dùng integer chứ
https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476#:~:text=In%20Ruby%20on%20Rails%2C%20an,for%20enums%20in%20Rails%204.1.

Reply:
status có cần default không vậy ???